### PR TITLE
[FEATURE] Custom title 

### LINKED ISSUE #964: [FEATURE] Custom Title
### Description

Hey there,
             Can we change the title dynamically. Like if someone lands on the profile page then the title would be something related to the Profile itself. 

For Example:    'Profile name' | LinkFree

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by aliraza944**
## Fixes Issue
Closes #964 

## Changes proposed
Dynamically changing the document title on the user profile page.

The profile document title is changing based on the name of the user
- Added document.title to the useEffect of User.js file
- The user name will be displayed on the document title
- Added the cleanup function to revert back to the original document title

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

#### Before changes
before

#### After changes
document title

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/968"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Could you remove the package.json and package-lock.json files from your commit and also your profile? The profile can be added as a separate pull request so that we can keep each pull request to one change only. Thanks 😊

**Review by schmelto — CHANGES_REQUESTED** [review]
Please don't delete the package.json and package-lock.json file. Only do not commit any changes made there to the PR.
Your PR should only contain changes made in `src/Components/UserPRofile/User.js`.

**Review by KendallDoesCoding — APPROVED** [review]
this is a good idea!

**Comment by emmalearnscode** [comment]
Unfortunately deleting the files causes problems, either revert your commit history and only add the file containing the custom title, or copy-paste the package-lock and page.json files from the main repo back into your fork and commit them.

**Comment by emmalearnscode** [comment]
Do you need any help @aliraza944 ?

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "linkfree",
-  "version": "0.59.0",
+  "version": "0.62.3",
   "private": false,
-  "homepage": "http://linkfree.eddiehub.org/",
+  "homepage": "http://linkfree.eddiehub.org",
```
Comment (schmelto) [inline package.json:5]:
```suggestion
  "homepage": "http://linkfree.eddiehub.org/",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "linkfree",
-  "version": "0.58.1",
+  "version": "0.62.3",
```
Comment (schmelto) [inline package-lock.json:3]:
```suggestion
  "version": "0.58.1",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.58.1",
+      "version": "0.59.1",
```
Comment (schmelto) [inline package-lock.json:9]:
```suggestion
      "version": "0.58.1",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by krshdev — CHANGES_REQUESTED** [review]
## Merge Conflict

Thank you for your contribution. Your pull request currently has conflicting changes with our `main` branch. In order to review this pull request, we will need those conflicts to be resolved.

We recommend doing this with the `git rebase` command. If you are unsure of how to fix this, let us know and we would be happy to assist you.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request closed without merging**